### ISSUE TITLE
aws_nat_gateway does not accept tags

### ISSUE THREAD

**Issue by nbirnel** [terraform-providers/terraform-provider-aws#1755]
### Terraform Version
Terraform v0.10.6

### Terraform Configuration Files
```hcl
resource "aws_nat_gateway" "nat_gateway_c" {
  allocation_id = "${aws_eip.nat-c.id}"
  subnet_id     = "${aws_subnet.public-sn-c.id}"

  depends_on = ["aws_internet_gateway.cf1-east"]

  tags {
      Name = "nat-a"
      env = "prod"
      terraform = "yes"
  }
}

```

### Output
1 error(s) occurred:

* aws_nat_gateway.nat_gateway_c: : invalid or unknown key: tags

### Expected Behavior
successful plan. AWS NAT gateways do accept tags through the web UI and through the aws cli.

### Steps to Reproduce
terraform plan --target=aws_nat_gateway.nat_gateway_c --out=tag-nat-c.plan

**Comment by stack72** [comment]
Hi @nbirnel 

The code for tagging NAT Gateways was just added in #1625

Is it waiting release in 1.0.0

Paul

**Comment by tombuildsstuff** [comment]
Hey @nbirnel 

This has been released in v1.0.0 of the AWS Terraform Provider - as such I'm going to close this out

Thanks!
